biz.aQute.repository/src/aQute/bnd/repository/maven/provider/Configuration.java

```diff
@@ -71,4 +71,10 @@
 	 * Extensions for files that contain multiple JARs
 	 */
 	String multi();
+
+	/**
+	 * Forces release of source and javadoc artifacts even if no snapshot or
+	 * release Url is set.
+	 */
+	boolean forceFullLocalRelease();
```

Comment:
I don't see this a repository configuration thing. It is really more something that should be on `-maven-release`.  Something like:

```properties
-maven-release: sources;local=true,javadoc;local=true
```

Then it can easily be configured on a per project basis rather than over the whole workspace.